20_2 DataGrid Master-Detail API demo: rewrite (#609)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/MasterDetailAPI/description.md

> DataGrid API allows you to implement a custom expand/collapse logic for master-detail rows. In this example, you can expand only one row at a time. To achieve this, we used the [expandRow(key)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#expandRowkey) and [collapseAll(groupIndex)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#collapseAllgroupIndex) methods inside the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) event handler.

Reviewer: ```suggestion
DataGrid API allows you to implement custom expand/collapse logic for master-detail rows. This example uses the [expandRow(key)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#expandRowkey) and [collapseAll(groupIndex)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#collapseAllgroupIndex) methods inside the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) event handler to allow you to expand only one row at a time.
```

> The DataGrid ships with an API that allows you to implement a custom expand/collapse logic for master-detail rows. In this example, you can expand only one row at a time. To achieve this, we used the [expandRow(key)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#expandRowkey) and [collapseAll(groupIndex)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#collapseAllgroupIndex) methods inside the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) event handler.

Reviewer 2: I'd only simplify the first part a bit: 

```suggestion
DataGrid API allows you to implement a custom expand/collapse logic for master-detail rows. In this example, you can expand only one row at a time. To achieve this, we used the [expandRow(key)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#expandRowkey) and [collapseAll(groupIndex)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#collapseAllgroupIndex) methods inside the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) event handler.
```
